CONTRIB.rst

```diff
@@ -157,6 +156,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - Jose Blanca <https://github.com/JoseBlanca>
 - Joshua Meyers <https://github.com/JoshuaMeyers>
 - João D Ferreira <https://github.com/jdferreira>
+- João Rodrigues <https://github.com/joaorodrigues>
```

Comment:
Off topic, but would changing the sorting rule to UTF-8 work better for the current set of international names? See head of file and style check via ``.travis-tox.ini`` doing:

```
$ grep "^- " CONTRIB.rst | LC_ALL=C sort -u -c -f
```

Reply:
Yeah, I found that weird too but it was a minor thing. Changing the sorting to `grep "^- " CONTRIB.rst | LC_ALL=en_US.UTF-8 sort -u -f -c` seems to order things correcty, but would this be available everywhere?

Reply:
I don't know - but as long as it works on our main developer's machines and TravisCI, that might be OK?

Reply:
I guess so - personally, I got used to not using the ~ in code as it gives too much trouble! I'll open an issue and tag some of the main devs to ask.